text2graph/alignment.py

```diff
@@ -16,7 +18,15 @@ def __init__(
         self.known_entity_names = known_entity_names
         self.known_entity_embeddings = known_entity_embeddings
 
-        self.model = SentenceTransformer(model_name)
+        if os.environ["ALIGNMENT_ENTITY_EMBEDDING_PROCESSOR"]:
+            device = os.environ["ALIGNMENT_ENTITY_EMBEDDING_PROCESSOR"]
+        else:
+            if torch.cuda.is_available():
+                device = "cuda"
+            else:
+                device = "cpu"
```

Comment:
happy for you to force to cpu. I put these in here so I could avoid sentence transformers using my mac's mps by default (since my mac's mps uses an AMD GPU, which doesn't work).